### PR TITLE
ODC-7527: Add repository details page

### PR DESCRIPTION
**Pull request by vikram-raj**

### CODE REVIEW

**Comment by vikram-raj** [comment]
> image
> Until I select Edit Repository button in the action menu I am able to see the blank screen in all 3 tabs. If I select Edit in action menu then after all 3 tabs are visible.

@lokanandaprabhu Fixed it.

File: src/components/pipelineRuns-list/PipelineRunsList.tsx [hunk 1]
```diff
@@ -70,6 +76,7 @@ const PipelineRunsList: React.FC<PipelineRunsListProps> = ({
         rowData={{
           taskRuns,
           taskRunsLoaded,
+          repositoryPLRs,
```
Comment (lokanandaprabhu) [inline src/components/pipelineRuns-list/PipelineRunsList.tsx:79]: Screenshot 2024-04-09 at 6 06 46 PM

For some reason top spacing is removed again

Comment (vikram-raj) [reply src/components/pipelineRuns-list/PipelineRunsList.tsx:79]: Fixed it.

**Review by lokanandaprabhu — APPROVED** [review]

**Pull request merged**